### DISCUSSION TITLE
Some compile error,why?please tell me

### DISCUSSION THREAD

**Discussion by 652889849** [LSPosed/LSPosed#1905] [Q&A]
C:\Work\GIT\LSPosed\LSPosed>gradlew.bat zipAll
Picked up _JAVA_OPTIONS: -Xmx1024M
Type-safe project accessors is an incubating feature.

> Configure project :app
WARNING:The option setting 'android.experimental.enableNewResourceShrinker.preciseShrinking=true' is experimental.
The current default is 'false'.
WARNING:The option setting 'android.enableAppCompileTimeRClass=true' is experimental.
The current default is 'false'.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':magisk-loader:processRiruDebugMainManifest'.
> Could not resolve all task dependencies for configuration ':magisk-loader:RiruDebugRuntimeClasspath'.
   > Could not resolve project :services:xposed-service:interface.
     Required by:
         project :magisk-loader > project :services:manager-service
      > No matching configuration of project :services:xposed-service:interface was found. The consumer was configured to find a runtime of a component, preferably optimized for Android, as well as attribute 'com.android.build.api.attributes.BuildTypeAttr' with value 'debug', attribute 'com.android.build.api.attributes.ProductFlavor:api' with value 'Riru', attribute 'com.android.build.api.attributes.AgpVersionAttr' with value '7.1.3' but:
          - None of the consumable configurations have attributes.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s

----
or have any docs to learn,thx.

**Comment by yujincheng08** [comment]
`git submodule update --init --recursive`
